cmd/skyeye/main.go

```diff
@@ -131,6 +145,37 @@ func main() {
 	}
 }
 
+func initializeConfig(cmd *cobra.Command) error {
+	v := viper.New()
+
+	v.SetConfigFile(configFile)
+	if err := v.ReadInConfig(); err != nil {
+		// having no config file is fine
+		if _, ok := err.(viper.ConfigFileNotFoundError); !ok {
```

Comment:
Use [errors.Is](https://pkg.go.dev/errors#Is) to check error types - it handles wrapped and joined errors correctly.

Reply:
done. hopefully its ok now :)